projects/storefrontlib/src/lib/ui/layout/landing-page-layout/landing-page-layout.component.ts

```diff
@@ -1,9 +1,10 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'cx-landing-page-layout',
   templateUrl: './landing-page-layout.component.html',
-  styleUrls: ['./landing-page-layout.component.scss']
+  styleUrls: ['./landing-page-layout.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
Is the none encapsulation really needed? I tested it with Emulated which is the default but I didn't see any changes. Perhaps I am missing something.